docs/build/blast-install-1.md

```diff
@@ -39,31 +29,13 @@ Checkout [Setup Rust](/docs/build/setup-rust)
 
 ## 02 Install Cudos Blast
 
-### Globally - Create a new project anywhere
-
-Use `npm` to install Cudos Blast CLI tool globally. This means you can create a new project anywhere. 
-
-```bash
-npm install cudos-blast -g
```

Comment:
Note that you will usually need to run this as root, so this should probably be written as `sudo npm install cudos-blast -g`

Reply:
Done